lib/filterx/expr-null-coalesce.c

```diff
@@ -46,11 +46,23 @@ _eval_null_coalesce(FilterXExpr *s)
     {
       if (!lhs_object)
         {
-          msg_debug("FILTERX null coalesce supressing error",
-                    filterx_expr_format_location_tag(s),
-                    filterx_eval_format_last_error_tag());
+          if (debug_flag)
+            {
+              gint error_count = filterx_eval_get_error_count();
+              gchar buf[FILTERX_EVAL_ERROR_IDX_FMT_SIZE];
+
+              for (gint err_idx = 0; err_idx < error_count; err_idx++)
+                {
+                  msg_debug("FilterX: null coalesce suppressing error",
+                            filterx_eval_format_error_index_tag(err_idx, buf),
+                            filterx_eval_format_error_location_tag(err_idx),
+                            filterx_eval_format_error_tag(err_idx));
+                }
+            }
+
```

Comment:
I feel a bit bad about the open-coded loop here as I am sure it will get repeated a number of times further down.